Library/Homebrew/os/linux/elf.rb

```diff
@@ -126,7 +132,22 @@ def initialize(path)
 
     private
 
+    def default_on_exception(val, exception)
+      yield
+    rescue exception
+      val
+    end
+
     def needed_libraries(path)
+      # adding support for HOMEBREW_PATCHELF_RB here, should be a simple conditional.
+      if HOMEBREW_READELF_RB
```

Comment:
Can we make this an `if...else` instead? That way each branch can use an implicit return and the control flow will be more obvious.

Reply:
does this look okay?

Reply:
Does to me 👍

Reply:
LGTM too!